config/routes.rb

```diff
@@ -0,0 +1,13 @@
+Rails.application.routes.draw do
+  namespace :api do
+    namespace :v1 do
+      get 'messages/index'
+    end
+    get 'v1/message'
```

Comment:
- [ ] Kindly remove this route as it doesn't have any use here. This is for a cleaner and more optimized code.